lib/rules/no-else-return.js

```diff
@@ -88,6 +102,17 @@ module.exports = {
                     }
 
                     const endToken = sourceCode.getLastToken(node);
+
+                    /**
+                     * If else block includes block scope local variable declaration [let or const],
+                     * then it is not safe to remove else keyword [issue 11069]
+                     */
+                    const isDeclarationInside = sourceCode.getTokensBetween(startToken, endToken).findIndex(isVariableDeclaration) > -1;
```

Comment:
Nit: You could use `.some` here to simplify

Reply:
@platinumazure  I've used .findIndex, because  it will stop iteration if it will find an element and .some method will iterate through whole  array, even it will find an element

Reply:
@g-plane you are right, thank you. I will change it!